lib/wcc/contentful/store/postgres_store.rb

```diff
@@ -5,7 +5,7 @@
 
 module WCC::Contentful::Store
   class PostgresStore < Base
-    def initialize(connection_options = nil)
+    def initialize(config = nil, connection_options = nil)
```

Comment:
are you using this config anywhere?

Reply:
@jpowell no i do not use it anywhere, did not know if in the future

Reply:
should be `_config` since unused